fixtures/007/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Layer version as string instead of int",
+  "specification_reference": "https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md",
+  "validity": {
+    "v1": true,
```

Comment:
@GretaCB I think these should be `false` since this is technically an invalid fixture